sci-libs/oce/oce-0.18.3.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils eutils check-reqs multilib java-pkg-opt-2
+
+DESCRIPTION="Development platform for CAD/CAE, 3D surface/solid modeling and data exchange"
+HOMEPAGE="https://github.com/tpaviot/oce"
+SRC_URI="https://github.com/tpaviot/oce/archive/OCE-$PV.tar.gz"
+
+LICENSE="|| ( Open-CASCADE-LGPL-2.1-Exception-1.0 LGPL-2.1 )"
+SLOT="${PV}"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="examples freeimage gl2ps openmp +tbb +vtk X"
```

Comment:
The `+tbb` default in conjunction with the `REQUIRED_USE` below makes this package conflict with the default profile which enables `openmp` by default. What's the implications of setting `+openmp` instead of `+tbb`?

Reply:
Also: any reason why `vtk` is enabled by default? It isn't enabled by default upstream and, according to your kicad ebuild, isn't required by its only revdep consumer. Not enabling it by default will save significant needless compilations for users who keep default USE flag values.

Reply:
I just assumed +tbb for whatever reason, but going through the docs I don't see why that should be the case. I'll flip it over to +openmp by default.

I +vtk inherited from the source ebuild, but I can rebuild and test without it and see if KiCad compiles and runs.